Tests/GitHubReactions.Tests.ps1

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+   Tests for GitHubReactions.ps1 module
+#>
+
+# This is common test code setup logic for all Pester test files
+$moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
+. (Join-Path -Path $moduleRootPath -ChildPath 'Tests\Common.ps1')
+
+try
+{
+    # Define Script-scoped, readonly, hidden variables.
+    @{
+        defaultIssueTitle = "Test Title"
+        defaultCommentBody = "This is a test body."
+        defaultReactionType = "+1"
+        otherReactionType = "eyes"
+    }.GetEnumerator() | ForEach-Object {
+        Set-Variable -Force -Scope Script -Option ReadOnly -Visibility Private -Name $_.Key -Value $_.Value
+    }
+
+    Describe 'Creating, modifying and deleting comments' {
```

Comment:
It would be great if these tests were also validating the additional Type and Properties that are expected to be added when calling `Get/Set/New`.  You are checking for `RepositoryUrl`, but you're also adding others too.

Reply:
added